pkg/aws/validations/iam_helpers.go

```diff
@@ -3,28 +3,31 @@ package validations
 import (
 	"fmt"
 
-	iamtypes "github.com/aws/aws-sdk-go-v2/service/iam/types"
 	"github.com/aws/aws-sdk-go-v2/aws"
+	iamtypes "github.com/aws/aws-sdk-go-v2/service/iam/types"
 	semver "github.com/hashicorp/go-version"
-	"github.com/openshift-online/ocm-common/pkg"
+)
+
+const (
+	maxByteSize = 64
```

Comment:
why was MaxByteSize replaced and moved here?

Reply:
Thought of making it closer to it's usage. Also pkg wasn't much clear as a package name :thinking: 
I did end up making it private though, and because it's used in rosa it cannot be